package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "helm",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@actions/core": "^1.2.6",
+    "@actions/exec": "^1.0.4",
+    "@actions/github": "^4.0.0",
+    "common-tags": "^1.8.0"
```

Comment:
Main thing I would say is to add prettier to ensure consistent formatting in regards to spaces and semicolons. But that isn't needed for an MVP.

Reply:
I added prettier, any tips for rules?